src/github_refresh.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+"""Script for refreshing GitHub data of ingested packages."""
+
+import os
+import logging
+from selinon import run_flow_selective
+from datetime import datetime, timedelta
+from f8a_worker.setup_celery import init_celery, init_selinon
+
+import botocore
+import boto3
+import json
+
+
+_REGION = os.environ.get('AWS_S3_REGION') or 'us-east-1'
+_ACCESS_KEY_ID = os.environ.get('AWS_S3_ACCESS_KEY_ID') or None
+_ACCESS_KEY = os.environ.get('AWS_S3_SECRET_ACCESS_KEY') or None
+_PREFIX = os.environ.get('DEPLOYMENT_PREFIX') or 'dev'
+_BUCKET = os.environ.get('REPORT_BUCKET_NAME') or None
+
+TASK_NAMES = [
+    'github_details',
+    'PackageFinalizeTask',
+    'PackageResultCollector',
+    'PackageGraphImporterTask'
+]
+
+session = boto3.session.Session(
+    aws_access_key_id=_ACCESS_KEY_ID,
+    aws_secret_access_key=_ACCESS_KEY,
+    region_name=_REGION)
+s3_resource = session.resource('s3', config=botocore.client.Config(signature_version='s3v4'))
+eco_list = ['npm', 'maven', 'pypi']
+logger = logging.getLogger(__file__)
+logging.basicConfig(level=logging.INFO)
+
+
+def retrieve_dict(object_key):
+    """Retrieve a dictionary stored as JSON from S3."""
+    return json.loads(retrieve_blob(object_key).decode())
+
+
+def retrieve_blob(object_key):
+    """Retrieve remote object content."""
+    return s3_resource.Object(_BUCKET, object_key).get()['Body'].read()
+
+
+def get_epv_list():
+    """Fetch the epvs from the daily stack report."""
+    if _ACCESS_KEY_ID is None or _ACCESS_KEY is None or \
```

Comment:
You can use python's `any` or `all` - https://www.geeksforgeeks.org/any-all-in-python/